tests/Networking/NexusMods.Networking.Downloaders.Tests/DownloadServiceDataStoreTests.cs

```diff
@@ -35,11 +35,12 @@ public DownloadServiceDataStoreTests(DownloadService downloadService, IDataStore
         _httpDownloader = httpDownloader;
     }
 
+    /*
```

Comment:
We should decide what to do with this file. Makes no sense to keep commented out code around.
I think we should implement regression tests for behaviors exposed in public interfaces. 

Since we are talking about a downloader this might require require some work to have a proper testing setup, either with a fake local download or by mocking out some select parts.

Since this is the second time Pause, Cancel and throughput have seen regressions I think it the tests would be worth it.